drivers/AnalogOut.h

```diff
@@ -138,7 +138,8 @@ class AnalogOut {
 
     virtual ~AnalogOut()
     {
-        // Do nothing
+        // deinitialize the pin configuration totally. This should allow chaining the definition of the pin
+          analogout_free(&this->_dac);
```

Comment:
please fix the code style - see travis astyle failure

Reply:
Sorry, But this is my first pull request. Please feel free to change my patch to whatever works for you by that I learn also to not make that mistake again. 
thanks.

Reply:
No worries, worth reading: https://os.mbed.com/docs/mbed-os/v5.13/contributing/style.html

Reply:
Can you please align the comment and the code line 143? Otherwise looks good

Reply:
Dear Martin, 
I cannot find this note as you mentioned (" Note: This is not currently used in the mbed-drivers")
Please write down for me the link to the file. 
Notice that My change is only for NUCLEO-L476RG